tests/acceptance/RobotLab/01_IPython_Notebook.robot

```diff
@@ -11,18 +11,35 @@ Firefox
     headlessfirefox
 
 *** Keywords ***
+Run Selenium keyword and return status
+    [Documentation]
+    ...  Run Selenium keyword (optionally with arguments)
+    ...  and return status without screenshots on failure
+    [Arguments]  ${keyword}  @{arguments}
+    ${tmp}=  Register keyword to run on failure  No operation
+    ${status}=  Run keyword and return status  ${keyword}  @{arguments}
+    Register keyword to run on failure  ${tmp}
+    [Return]  ${status}
+
 Can RobotLab make an IPython Notebook?
     [Arguments]    ${browser}
     [Documentation]    Try a basic IPython Notebook
     Set Tags    browser:${browser}
     ${prefix} =    Set Variable    ipython_${browser}_
     Open JupyterLab    browser=${browser}
+    ${is_launcher}=  Run Selenium keyword and return status
+    ...  Page should contain element
+    ...  xpath://div[@class='jp-LauncherCard']
+    Run keyword if  not ${is_launcher}
+    ...  Execute JupyterLab Command  New Launcher
```

Comment:
@bollwyvl This is probably something to be taken account in JupyterLibrary. The logs did show that tests failed, because JupyterLab did start with empty Untitled.ipynb instead of the launcher. I assume that's because of some state being left between tests into the local filesystem.

Reply:
K, thanks. I should probably look into creating new workspaces for each test. Heck, since they can be serialized, maybe it would make sense to have...
```robot
*** Keywords ***
open to jupyterlab workspace
  [arguments]  &{widgets}

*** Test Cases ***
open the launcher
   open jupyterlab workspace  Launcher=

open a notebook and stuff
   open jupyterlab workspace  
   ...  Notebook:Test Case 1.ipynb    
   ...  Terminal    
   ...  Inspector  
   ...  left=Cell Tools
```

Reply:
Or, of course,
```robot
open a notebook and stuff
   open jupyterlab workspace    file=test1.workspace.json
```